threatstream.json

```diff
@@ -27232,4 +27225,4 @@
             }
         ]
     }
-}
+}
```

Comment:
It was good with the `\n`

Reply:
It's the case of my autoformatter to remove \n, can it stay that way?

Reply:
It theoretically does no harm but it's one of those good practices that pretty much everyone follows, based on unix's definition of a line, better fix your autoformatter